regipy/plugins/system/bam.py

```diff
@@ -31,7 +31,7 @@ def run(self):
                     version = None
                     entries = []
 
-                    for value in sid_subkey.get_values():
+                    for value in sid_subkey.get_values(trim_values=self.trim_values):
```

Comment:
You do not want to set `trim_values` as `True` for this specific plugin?

Reply:
Thanks!!